draft-ietf-acme-acme.md

```diff
@@ -303,12 +303,21 @@ JWS objects sent in ACME requests MUST meet the following additional criteria:
 * The JWS MUST NOT have the value "none" in its "alg" field
 * The JWS Protected Header MUST include the following fields:
   * "alg"
-  * "jwk"
+  * "jwk" (only for requests to new-reg and revoke-cert resources)
+  * "kid" (for all requests except for new-reg and revoke-cert requests).
   * "nonce" (defined below)
   * "url" (defined below)
 
-Note that this implies that GET requests are not authenticated.  Servers MUST
-NOT respond to GET requests for resources that might be considered sensitive.
+For new-reg requests and revoke-cert-by-key requests, there MUST be a
+"jwk" field containing the key being registered, and there MUST NOT be a
+"kid" field.
+
+For all other requests, there MUST be a "kid" field containing the account URI
+received by POSTing to the new-reg resource.
```

Comment:
You still need to say what the server does when it receives a request with any combination of `kid` and `jwk`.  There are two possibilities:

1. Require the server to reject requests that don't meet `(kid && !jwk) || (!kid && jwk)`
2. Require the server to ignore `jwk` when `kid` is present.

I don't much care which.